node/subsystem-util/src/validator_discovery.rs

```diff
@@ -99,14 +116,16 @@ pub async fn connect_to_past_session_validators<Context: SubsystemContext>(
 async fn connect_to_authorities<Context: SubsystemContext>(
 	ctx: &mut Context,
 	validator_ids: Vec<AuthorityDiscoveryId>,
+	peer_set: PeerSet,
 ) -> mpsc::Receiver<(AuthorityDiscoveryId, PeerId)> {
-	const PEERS_CAPACITY: usize = 8;
+	const PEERS_CAPACITY: usize = 32;
```

Comment:
Why was that increase needed?

Reply:
Was this causing the timeout in the test?

Reply:
It was not needed, but the initial capacity was chosen quite arbitrarily and is low. We will drop response records when the buffer is full, so it's better be high enough.